src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -288,7 +310,9 @@ internal override string GetToolName()
         /// <param name="consoleOutput"></param>
         public override void PrintHelp(TextWriter consoleOutput)
         {
-            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture));
+            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture),
+                LanguageVersion.Default.MapSpecifiedToEffectiveVersion().ToDisplayString(),
+                LanguageVersion.Latest.MapSpecifiedToEffectiveVersion().ToDisplayString());
```

Comment:
I think it's better if they are the latest versions, as this gives you an idea of the allowable range. Also it gives information about the most common case without having to invoke another command (`/langversion:?`). The funny part is giving examples that are less relevant and getting outdated. 

The complication is trivial, so I'm surprised you're pushing back. We use parameterized outputs all the time.
What are you proposing?